cookbooks/stac-zarr-report/data-consumers/index.qmd

```diff
@@ -0,0 +1,90 @@
+---
+title: "Data Consumers"
+subtitle: "Accessing data cubes in Zarr + STAC"
+---
+
+This section is targeted at people who are trying to access data cubes in Python. It addresses questions like:
+ - How do you find the data you need?
+ - How do you filter down to a subset of the data?
+ - When should you write to a virtualized file?
+
+This section assumes a world in which you as a data consumer are interacting with Zarr stores cataloged by STAC.
+
+## How to read the minimum amount of data
+
+One of the benefits of using a cloud-optimized file format like Zarr is that you don't have to read all the data in a dataset. You can filter the data to just the variables, area and time period that you are interested in. The sooner you do this filtering the quicker the rest of the workflow will be.
+
+Depending on your background you might have a preference for interacting with the entire lazily-loaded data cube in xarray. Or you might want to use STAC tooling to do searching and filtering on the server-side and only start interacting with the data cube after filtering has been applied. 
+
+Both of those access patterns should be supported by tooling, but depending on how the catalog is set up some patterns may be simpler and faster than others (at this point in time). So which is better? Basically it depends on the structure of the Zarr store(s) and the STAC catalog that you are interacting with.
+
+## One big Zarr store in a standalone STAC collection
+
+The STAC catalog contains a collection for each Zarr store and there are collection-level assets that point to the location of the Zarr store. There are no items at all -- or if there are items they have no assets.
+
+In this scenario any STAC metadata exists purely for discovery and cannot be used for filtering or  subsetting (see [Future Work](#future-work) for more on that). To search the STAC catalog to find collections of interest you will use the [Collection Search API Extension][7] (TODO: check if a static version has been implemented in stac-rs). Depending on the level of metadata that has been provided in the STAC catalog you can search by the name of the collection and possible by the variables (aka groups) contained within the Zarr store.
```

Comment:
> TODO: check if a static version has been implemented in stac-rs)

It hasn't, but I've opened a tracking issue: https://github.com/stac-utils/rustac/issues/704